lib/pasarguard-backup.sh

```
@@ -272,78 +291,82 @@ send_backup_to_telegram() {
+        info_message+=$'🪟 Windows: Install and use 7-Zip. Place the .zip and every .zXX part together, then start extraction from the .zip file.\n'
+        info_message+=$'🐧 Linux: Run unzip (e.g., unzip backup_xxx.zip) with all .zXX parts in the same directory.\n'
+        info_message+=$'🍎 macOS: Use Archive Utility or run unzip backup_xxx.zip from Terminal with the .zXX parts beside the .zip file.\n'
+        info_message+=$'⚠️ Always download the .zip and every .zXX part before extracting.'
+    else
+        info_message+=$'🪟 Windows: Extract the uploaded archive with 7-Zip.\n'
+        info_message+=$'🐧 Linux: Run unzip backup_xxx.zip.\n'
+        info_message+=$'🍎 macOS: Open the archive with Archive Utility or run unzip backup_xxx.zip.\n'
+        info_message+=$'⚠️ Download the complete archive before extracting.'
+    fi
+
+    local escaped_info_message=$(escape_markdown_v2 "$info_message")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)